mozilla/addons-server
# Re-implement cinder forwards (escalations) as a new job in the content infringement queue

Fixes: mozilla/addons#14970



### Description

Reimplement forwards from cinder (nee escalations) without creating a CinderDecision, and creates a new CinderJob in the correct queue to track it. 

### Context

The renaming to forwarding to escalations was a low-priority Operations request that seemed easy to do at the same time.

More context in the issue and parent story ticket.  Essentially, forwarded jobs will be implemented by ~moving into~ _recreating the job in_ the special queue on cinder where AMO handled reviews are kept.

~The webhook implementation itself will be in a separate issue~ _There is no need for any new webhook implementation at this time.  We re-create the job in the content infringement queue ourselves.  There may be a future step where we re-implement this re-create step as a workflow in Cinder, and monitor a webhook instead._

~(The 3rd step will be to remove support for the AMO_ESCALATE_ADDON entirely, once T&S have updated whatever operational/training docs they have that references escalations)~

### Testing

The functionality should be the same as now, with any existing jobs that are escalated migrated.
Setup, before checking out the patch:
- Either escalate an existing job from cinder and apply the webhook JSON locally; or manually create the appropriate CinderDecision instance (`CinderDecision(action=AMO_ESCALATE_ADDON, notes='xxx', ...)`, update the job to be `handled_by_reviewers=True`) for a cinder handled job.

Then:
- ~run the migrations, see the CinderDecisions are deleted and the CinderJobs updated to `is_forwarded=True`, with the associated notes.~
  - ~bonus: reverse the migration, and re-apply it.~
- Escalate a different existing job from cinder and apply the webhook JSON locally
- See the job shows correctly as forwarded in the reviewer tools
- See the job has been recreated in cinder, in the content infringement queue

Footguns:
- make sure the waffle switch for escalations is enabled
- make sure the add-on you report has an is_signed=True current_version
### Checklist
